src/benchmarks/micro/Serializers/Binary_FromStream.cs

```diff
@@ -37,6 +37,7 @@ public void SetupBinaryFormatter()
 
         [BenchmarkCategory(Categories.Libraries)]
         [Benchmark(Description = nameof(BinaryFormatter))]
+        [MemoryRandomizationAttribute]
```

Comment:
Why is there `Attribute` word in this? Why is it not just `MemoryRandomization`?

Reply:
I just mis-read the documentation. Forgot that you don't include the Attribute part of the name for attributes. Also fixed up the extra `#if` errors.